development/extensions/tutorial_testing.rst

```diff
@@ -953,15 +960,63 @@ the phpBB installation from GitHub for us:
         $ cd path/to/your/extension
         $ git update-index --chmod=+x travis/prepare-phpbb.sh
 
+Preparing EPV
+---------------
+
+The third file we need to create is a helper file called
+``travis/prepare-epv.sh``, which is a script used by Travis CI, to set up
+the phpBB extension prevalidator (EPV) for us:
+
+.. warning::
+
+    You should not edit the content of this file!
+
+.. code-block:: bash
+
+    #!/bin/bash
+    #
+    # This file is part of the phpBB Forum Software package.
+    #
+    # @copyright (c) phpBB Limited <https://www.phpbb.com>
+    # @license GNU General Public License, version 2 (GPL-2.0)
+    #
+    # For full copyright and license information, please see
+    # the docs/CREDITS.txt file.
+    #
+    set -e
+    set -x
+
+    EPV=$1
+    NOTESTS=$2
+
+    if [ "$EPV" == "1" ] && [ "$NOTESTS" == "1" ]
+    then
+        cd phpBB
+        composer remove sami/sami --update-with-dependencies --dev --no-interaction
+        composer require phpbb/epv:dev-master --dev --no-interaction --ignore-platform-reqs
+        cd ../
+    fi
+
+.. note::
+
+    The prepare-phpbb.sh file needs to have executable permissions or Travis CI
+    tests will fail. You can set the correct permission for this file from a
+    terminal command line interface, e.g.
+
+    .. code-block:: bash
+
+        $ cd path/to/your/extension
+        $ git update-index --chmod=+x travis/prepare-epv.sh
+
```

Comment:
Please remove all of this. It's too overwhelming. The command originally in .travis scripts section is ecaxtly the same, and much easier than asking users to create more files and worry about making them executable.

Also, keeping this as it is then mirrors what the skeleton extension outputs so we dont further confuse developers.